components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -168,14 +181,27 @@ export default function ProjectSettingsView() {
                 />
                 {enablePrebuilds && (
                     <>
-                        <InputField label="Workspace machine type" disabled={!enablePrebuilds}>
-                            <div className="max-w-md">
-                                <SelectWorkspaceClassComponent
-                                    disabled={!enablePrebuilds}
-                                    selectedWorkspaceClass={project.settings?.workspaceClasses?.prebuild}
-                                    onSelectionChange={setWorkspaceClassForPrebuild}
-                                />
-                            </div>
+                        <SelectInputField
+                            disabled={!enablePrebuilds}
+                            label="Build branches"
+                            value={prebuildBranchStrategy}
+                            containerClassName="max-w-md ml-6 text-sm"
```

Comment:
**suggestion**: Not sure if this is the most relevant line, but regarding the indentation, this `ml-6` and `mt-4`.

<img width="1328" alt="Frame 1889" src="https://github.com/gitpod-io/gitpod/assets/120486/b398268f-ef67-43ef-a306-d3e0bd4f9351">